ansible/group_vars/all/main.yml

```diff
@@ -2,8 +2,29 @@
 #
 # Variables here are applicable to all host groups.
 #
-
 region: us-east-1
 
+proxy_env: 
+  http_proxy: "{{ vault_http_proxy }}" 
+  https_proxy: "{{ vault_https_proxy }}"
+
+test_lib: "../target/fhir-stress-test-0.1.0-SNAPSHOT-uber.jar"
+test_plan: "../jmeter-project/jmeter-fhir-test.jmx"
+test_keystore: "../dev/ssl-stores/client.keystore"
+remote_test_dir: "/opt/fhir_stress"
+jmeter_version: 3.3
+remote_jmeter_dir: "{{ remote_test_dir }}/apache-jmeter-{{ jmeter_version }}"
+jmeter_server_port: "{{ vault_jmeter_server_port }}"
+jmeter_server_rmi_local_port: "{{ vault_jmeter_server_rmi_local_port }}" 
+jmeter_client_rmi_local_port: "{{ vault_jmeter_client_rmi_local_port }}"
+fhir_server: "https://fhir.backend.bluebutton.hhsdevcloud.us/baseDstu3"
+thread_count: 10
+thread_loops: 1000
+scheduler: true
+duration: 300
```

Comment:
Could you add a comment somewhere in here explaining the relationship between `thread_count`, `thread_loops`, and `duration`? They're all involved in how long the tests run and in how "hard" they'll run, right? But it's not clear here what you're trying to accomplish (because I can't remember how they interact myself).